shuffle/tailwind.config.js

```diff
@@ -1,19 +1,77 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  darkMode: ["class"],
```

Comment:
Is this what we want?

Reply:
Yes. this way we can have control over dark-mode with `dark:some-class` tailwind decorator.

Reply:
https://tailwindcss.com/docs/dark-mode To me it says the opposite. 😢 class mode is when you have to manually add class="dark" to your HTML if you want dark.